packages/components/src/MediaObject.tsx

```diff
@@ -0,0 +1,114 @@
+/** @jsxImportSource theme-ui */
+
+import { useState, useEffect } from 'react'
+import { useNFTContentType } from '@cura/hooks'
+import { Placeholder } from './Placeholder'
+
+type mediaObjectProps = {
+    mediaURI: string
+    width?: number | string
+    height?: number | string
+    loading?: boolean
+    autoPlay?: boolean
+}
+
+function Text({ mediaURI, width, height }: mediaObjectProps) {
+    const [content, setContent] = useState('')
+
+    useEffect(() => {
```

Comment:
In general, for the future, let's keep data retrieve logic either in the package that uses this component that would be `examples/frontend.` The goal of the `components` is only to have dump component (only data coming through props).